pkg/ingester/ingester_v2.go

```diff
@@ -24,7 +24,7 @@ import (
 
 	"github.com/cortexproject/cortex/pkg/ingester/client"
 	"github.com/cortexproject/cortex/pkg/ring"
-	cortex_tsdb "github.com/cortexproject/cortex/pkg/storage/tsdb"
+	cortex_tsdb "github.com/cortexproject/cortex/pkg/storage/backend"
```

Comment:
May you remove the package renaming to `tsdb` at all, please? I mean changing this line to:
`"github.com/cortexproject/cortex/pkg/storage/backend"`

And then fix the package references across the file.

_Same comment applies across the whole PR._